Automattic/pocket-casts-ios
# Ratings: add the structure for the give rating screen

Adds the initial structure for the give rating screen:

1. Feature flag
2. Screen and view model

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/53dd6814-a0c9-4106-951e-ca3499e2a6fc" width="300">

## To test

1. Run the app
2. Go to any podcast
3. Tap the stars
4. ✅ Nothing should happen
5. Go to Profile > Settings > Beta Features > enable `giveRatings`
6. Go back to the podcast
7. Tap the stars
5. ✅ A screen should appear
6. Tap dismiss
7. ✅ It should dismiss

## Checklist
